### PR TITLE
MoveMap Pam Scoring

### PR DESCRIPTION
**Pull request by mglubber**
Separate version of cas9_PAM_scoring with MoveMap and a couple of other features added

### CODE REVIEW

File: models/tridimensional/cas9_PAM_scoring_MoveMap.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        f.write(text)
+        text = "%8.3f," % (time_final- time_init)
+        f.write(text)
+        f.close()
+        print "DONE WRITING"
+
+if __name__ == '__main__':
+    # Uncomment the next two lines to parallize, can just pass any list to 
+    #p = Pool(processes=threads)
+    #p.map(scorer,listOfPAMsToRun)
```
Comment (Ellmen) [inline models/tridimensional/cas9_PAM_scoring_MoveMap.py:111]: I like this. If we stick with it I can add some functionality to also be able to run ranges. Maybe something like [1, [4,7],23] runs 1,4,5,6,7,23.

**Comment by jrhawley** [comment]
Tried to run it, it just crashed on me after 1 minute or two. Just got python.exe has stopped working, I'm not sure what the problem is. Did you get it to run properly on your computer @mglubber ?

**Comment by mglubber** [comment]
Hey,

Should have mentioned in the comments, it isn't running on my computer
either, I put it up so Matt could take a look. I had the same error, maybe
a windows thing?

Mark

**Comment by Ellmen** [comment]
I got the same error on Ubuntu so it's not a Windows thing. Mine also said it was a segmentation fault which might mean it's running out of memory. I'll try running it with more memory. Also someone mentioned on one of the forums that the Shear and Fragment Movers were buggy. Do you know if docking uses those?

**Comment by Ellmen** [comment]
Increasing the memory didn't seem to help

**Comment by mattsmart** [comment]
Chances are buggy c module within the rosetta calls. Is there any more info from the crash on linux?

**Comment by Ellmen** [comment]
No just "Segmentation fault (core dumped)" in the terminal. Could we try using minmover instead? It seems like it does the same thing.

**Comment by mglubber** [comment]
It works with min mover, I already tried it.

**Comment by mattsmart** [comment]
Merging to archive movemap

**Pull request merged**